engine/render/src/render/render.cpp

```diff
@@ -721,12 +721,17 @@ namespace dmRender
         params.m_Operation = RENDER_LIST_OPERATION_BEGIN;
         params.m_Context = context;
 
-        // All get begin operation first
-        for (uint32_t i=0;i!=context->m_RenderListDispatch.Size();i++)
         {
-            const RenderListDispatch& d = context->m_RenderListDispatch[i];
-            params.m_UserData = d.m_UserData;
-            d.m_DispatchFn(params);
+            DM_PROFILE("Dispatch_Begin");
```

Comment:
I don't know exactly how the DM_PROFILE generates the scope name, but 'dispatch' is a generic name so maybe we should say what we are dispatching, something like RenderListDispatch?

Reply:
The name is local, and it's displayed under the parent scope (currently "Render")